spec/integration/indirector/facts/facter_spec.rb

```diff
@@ -127,7 +127,7 @@
       it "prefers agent_specified_environment from agent if set in multiple sections" do
         set_puppet_conf(Puppet[:confdir], <<~CONF)
         [main]
-        serverport=baz
+        serverport=80
```

Comment:
crap, this should have been `environment=baz` but I changed the setting name to test something and forgot to change it back

Reply:
Fixed in https://github.com/puppetlabs/puppet/pull/8665 and https://github.com/puppetlabs/puppet/pull/8666